packages/tools/kadena-cli/src/utils/logger.ts

```diff
@@ -46,9 +48,10 @@ const LEVELS = {
   error: 0,
   warning: 1,
   output: 2,
-  info: 3,
-  debug: 4,
-  verbose: 5,
+  header: 3,
```

Comment:
@barthuijgen 

Could find a name, i needed a green color but it wasnt "success" any thoughts?
also we are now able to logTables, i would like to have your feedback

Reply:
I was also thinking about a `success` but struggled to put it in a level-order. Typically in logging libraries levels are used to be able to configure how much detail you want from the program ranging from fatal to verbose as a scale.

I took this over because of how common it is, and thought it would be useful for us as well.

But a `success` level-wise just feels like `info` to me, that's why I did not introduce it and opted for just using `log.color.green`, but I do understand having the ability to leave that out is a lot easier.

Adding more levels should not be a problem, I do think `success` still makes more sense then `header` as a level. 

We could also introduce new methods to the class that do not have their own levels, if we accept a 3rd argument like this:

```typescript
  public header(...args: unknown[]): void {
    this._log(LEVELS.info, args, COLORS.green);
  }
```

Reply:
got it , if we ever need it in the future we can still add it, since we now gave the tableheader a green color by default we don't need it now